pages/studentAssessmentInstanceExam/studentAssessmentInstanceExam.ejs

```diff
@@ -206,7 +206,7 @@
 
               <td class="text-center"><%- include('../partials/examQuestionScore', {instance_question}); %></td>
               <% if (assessment.allow_real_time_grading) { %>
-                <td class="text-center"><%- include('../partials/examQuestionAvailablePoints', {open: assessment_instance.open && instance_question.open, points_list: instance_question.points_list}); %></td>
+                <td class="text-center"><%- include('../partials/examQuestionAvailablePoints', {open: assessment_instance.open && instance_question.open, currentWeight: instance_question.points_list_original[instance_question.number_attempts], points_list: instance_question.points_list, highest_submission_score: instance_question.highest_submission_score}); %></td>
```

Comment:
This one had two changes -- I think there was a mistake in the hint text for scores > 0 on the overview page.  They were defaulting to `bestScore == 0`.

Reply:
Good catch!